.gitignore

```diff
@@ -0,0 +1 @@
+/nbproject
```

Comment:
I don't think I need to add an IDE-specific .gitignore file - if you are having a local problem with netbeans generating unwanted files, please add them to .git/info/exclude (which is not checked in as part of the repo).